.github/workflows/rust-base.yaml

```diff
@@ -72,6 +72,15 @@ on:
         required: false
         default: false
         type: boolean
+      cache-include-cargo-files:
+        description: 'Whether to include the Cargo toml and lock file in the cache key'
+        required: false
+        default: false
+        type: boolean
+      cache-save-branch-name:
```

Comment:
should we default this to main

Reply:
Fixed [b3e707a](https://github.com/phylaxsystems/actions/pull/33/commits/b3e707a70e18492b0cf19ae9e24f36bb81b5781b)